pkgs/development/libraries/dnnl/default.nix

```diff
@@ -0,0 +1,56 @@
+/*
+  This library can optionally use the MKL package. This is disabled by default
+  since it has a proprietary license; to enable it, add the following config
+  variable in an overlay:
+
+  self: super: {
+    mklSupport = true;
```

Comment:
I don't think we should encourage setting non-derivation top-level attributes. It is convenient though because `callPackage` picks it up.